lib/cc/analyzer/engines_builder.rb

```diff
@@ -44,7 +44,7 @@ def engine_config(raw_engine_config)
 
       def names_and_raw_engine_configs
         {}.tap do |ret|
-          @config.engines.each do |name, raw_engine_config|
+          (@config.engines || []).each do |name, raw_engine_config|
```

Comment:
Out of curiosity, is this for the case where the `engines` array is entirely missing from the YAML?